src/pycram/datastructures/world.py

```diff
@@ -128,51 +66,92 @@ class World(StateEntity, ABC):
      the URDF with the name of the URDF on the parameter server. 
     """
 
-    data_directory: List[str] = [os.path.join(os.path.dirname(__file__), '..', '..', '..', 'resources')]
+    cache_manager: CacheManager = CacheManager(conf.cache_dir, [conf.resources_path])
+    """
+    Global reference for the cache manager, this is used to cache the description files of the robot and the objects.
+    """
+
+    prospection_world_prefix: str = conf.prospection_world_prefix
+    """
+    The prefix for the prospection world name.
+    """
+
+    let_pycram_move_attached_objects: bool = conf.JobHandling.let_pycram_move_attached_objects
```

Comment:
The problem is that the values change depending on the type of world , the default values of world_conf.py are for Bulletworld, now if one uses Multiverse, the values that should be used are the ones in multiverse_conf.py, that is why these variables are needed because there values change by the world type.

Reply:
Done, I added a class in the config files, and also anywhere the configurations are used they should be used by accessing the instance of the world not like this "World.conf.." because this will access the incorrect conf if another type of world is being used.